desktop/src/main/java/bisq/desktop/main/funds/transactions/TransactionsView.java

```diff
@@ -251,6 +299,81 @@ protected void deactivate() {
         exportButton.setOnAction(null);
     }
 
+    private void updateList() {
+        List<TransactionsListItem> transactionsListItems = btcWalletService.getTransactions(false)
+                .stream()
+                .map(transaction -> {
+                    Set<Tradable> tradables = tradableRepository.getAll();
+
+                    TransactionAwareTradable maybeTradable = tradables.stream()
+                            .map(tradable -> {
+                                if (tradable instanceof OpenOffer) {
+                                    return new TransactionAwareOpenOffer((OpenOffer) tradable);
+                                } else if (tradable instanceof TradeModel) {
+                                    return new TransactionAwareTrade(
+                                            (TradeModel) tradable,
+                                            arbitrationManager,
+                                            refundManager,
+                                            btcWalletService,
+                                            pubKeyRing
+                                    );
+                                } else {
+                                    return new DummyTransactionAwareTradable(tradable);
```

Comment:
I know that is from old code, but as that case never gets executed I think we should return null here and then filter out null in the next step. Tradable is only extended from OpenOffer and TradeModel (one other case is not relevant here). The DummyTransactionAwareTradable can be removed then.